test/src/701-xattr-catalog_counters/main

```diff
@@ -0,0 +1,61 @@
+source ./src/701-xattr-catalog_counters/setup_teardown
+
+cvmfs_test_name="Extended attribute: catalog_counters"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+check_catalog_hashes() {
+  echo "Retrieve hashes from cvmfs_server list-catalogs"
+  local list_catalogs=`cvmfs_server list-catalogs -hx ${CVMFS_TEST_REPO}`
+  local hash_root=$(echo "$list_catalogs" | grep '/$' | awk -F' ' '{print $1}')
+  local hash_foo=$(echo "$list_catalogs" | grep '/foo$' | awk -F' ' '{print $1}')
+  local hash_bar=$(echo "$list_catalogs" | grep '/bar$' | awk -F' ' '{print $1}')
+ 
+  echo "Retrieve catalog hashes for files"
+  local catalog_hash_root1=$(attr -g catalog_counters /cvmfs/$CVMFS_TEST_REPO/root1.txt | grep 'hash' | awk -F' ' '{print $2}' ) 
+  local catalog_hash_root2=$(attr -g catalog_counters /cvmfs/$CVMFS_TEST_REPO/root2.txt | grep 'hash' | awk -F' ' '{print $2}' ) 
+
+  ## THIS IS NOT POSSIBLE BECAUSE ROOT DIR
+  # local catalog_hash_root_dir=$(attr -g catalog_counters /cvmfs/$CVMFS_TEST_REPO/ | grep 'hash' | awk -F' ' '{print $2}' ) 
```

Comment:
This should be possible... do you get an error?

Reply:
it is possible but the result is empty. with `grep` and `awk` behind it they start complaining about to work on an empty string